crypto/src/random/default.rs

```diff
@@ -208,7 +153,8 @@ impl<B: StarkField, H: ElementHasher<BaseField = B>> RandomCoin for DefaultRando
         Err(RandomCoinError::FailedToDrawFieldElement(1000))
     }
 
-    /// Returns a vector of unique integers selected from the range [0, domain_size).
+    /// Returns a vector of unique integers selected from the range [0, domain_size) after reseeding
+    /// the PRNG with the specified `nonce`.
```

Comment:
On the other hand, here I'd probably mention the exact methodology as this is a concrete implementation.

Reply:
Makes sense.